ros2_java_android.repos

```diff
@@ -2,80 +2,84 @@ repositories:
   eProsima/Fast-CDR:
     type: git
     url: https://github.com/eProsima/Fast-CDR.git
-    version: v1.0.7
+    version: release/1.0.10
   eProsima/Fast-RTPS:
     type: git
     url: https://github.com/eProsima/Fast-RTPS.git
-    version: 7a0b0fe7ca8d2c4ea41e36744c6024c263a6505a
+    version: release/1.8.0
```

Comment:
Looking at https://github.com/ros2/ros2/blob/dashing/ros2.repos , this should be the v1.8.2 tag.